examples/gno.land/p/demo/dao_maker/jsonutil/gno.mod

```diff
@@ -0,0 +1,7 @@
+module gno.land/p/demo/dao_maker/jsonutil
```

Comment:
The current `p/demo/json` library should be suitable for your needs.

I'm against having any JSON library written in userland because I expect that we can use @gfanton's #1776 and avoid any boilerplate code that isn't either for humans (rendering) or for other contracts (imports). Generally, anything about marshalling doesn't fall into these categories.

In the meantime, you can reuse the existing JSON library if needed. However, as soon as possible, I expect we can delete it completely. Additionally, if possible, I expect we don't merge any new JSON library in the meantime.

Reply:
this extends `p/demo/json`